go.sum

```diff
@@ -50,7 +50,6 @@ github.com/google/go-cmp v0.3.1/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMyw
 github.com/google/go-cmp v0.4.0 h1:xsAVV57WRhGj6kEIi8ReJzQlHHqcBYCElAvkovg3B/4=
 github.com/google/go-cmp v0.4.0/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/gNBxE=
 github.com/hpcloud/tail v1.0.0/go.mod h1:ab1qPbhIpdTxEkNHXyeSf5vhxWSCs/tWer42PpOxQnU=
-github.com/jessevdk/go-flags v1.4.0 h1:4IU2WS7AumrZ/40jfhf4QVDMsQwqA7VEHozFRrGARJA=
```

Comment:
Getting an error due to the removal of this line:
```
┌─[dependabot/go_modules/golang.org/x/net-0.7.0][~/workspace/concourse/docs]
└─▪ ./scripts/build -s 8000
../../../go/pkg/mod/github.com/vito/booklit@v0.12.0/booklitcmd/main.go:8:2: missing go.sum entry for module providing package github.com/jessevdk/go-flags (imported by github.com/vito/booklit/booklitcmd); to add:
        go get github.com/vito/booklit/booklitcmd@v0.12.0
```

Reply:
Running `go get github.com/vito/booklit/booklitcmd@v0.12.0` adds that line back and then everything works.

Reply:
Running `go mod tidy` on master also removes this line